src/Emanote/Model/Stork/Index.hs

```diff
@@ -79,8 +82,20 @@ runStork input = do
       -- title (but why would they?)
       T.replace "\\\\U" "\\U"
 
-newtype Input = Input
-  { inputFiles :: [File]
+data Input = Input
+  { inputFiles :: [File],
+    inputFrontmatterHandling :: Handling
+  }
+  deriving stock (Eq, Show)
+
+data Handling
+  = Handling_Ignore
+  | Handling_Omit
+  | Handling_Parse
+  deriving stock (Eq, Show)
+
+newtype StorkInput = StorkInput
+  { globalInput :: Input
```

Comment:
Maybe, `newtype Config` and `configInput` (to keep with the field naming convention).